packages/kit/src/core/create_manifest_data/index.js

```diff
@@ -238,9 +239,37 @@ export default function create_manifest_data({ config, output, cwd = process.cwd
 	walk(config.kit.files.routes, [], [], [layout], [error]);
 
 	const assets_dir = config.kit.files.assets;
+	/**
+	 * @type {import('types/internal').Asset[]}
+	 */
+	let assets = [];
+	if (fs.existsSync(assets_dir)) {
```

Comment:
it might be nice to move this into a separate function that returns the list of assets

Reply:
Done.